x-pack/test/cloud_security_posture_functional/config.ts

```diff
@@ -38,7 +38,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
          *   2. merge the updated version number change to kibana
          */
         `--xpack.fleet.packages.0.name=cloud_security_posture`,
-        `--xpack.fleet.packages.0.version=1.0.8`,
+        `--xpack.fleet.packages.0.version=1.2.11`,
```

Comment:
Every version is immutable. Whenever we introduce `next` version. It doesn't get updated from any followup change.
We will start using `preview` suffix, because it allows to continuously increment it (e.g., `preview1`, `preview2` etc, which with `next` you cannot do that)

So I don't expect any issue using prerelease versions